tests/pkg/mcufont/Makefile.ci

```diff
@@ -0,0 +1,4 @@
+BOARD_INSUFFICIENT_MEMORY := \
```

Comment:
Did you run `make -C tests/pkg/mcufont/ generate-Makefile.ci` to generate this list? (it looks too small)

Reply:
Did now (I always forget the existence of that target). List didn't change though, it is this small.